README.md

```diff
@@ -1,24 +1,25 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Hello World
 
-> My first Microverse project.
+## Portfolio. Project 1 : setup and mobile version skeleton
+
+>  My first milestone in the process of creating my portfolio website.
 
 ## Built With
 
-- HTML, CSS
+-  HTML, CSS
 
 ## Authors
 
 👤 **Zermarden**
 
-- GitHub: [@nedramrez](https://github.com/nedramrez)
-- Twitter: [@zermarden](https://twitter.com/zermarden)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/zermarden)
+-  GitHub: [@Nedramrez](https://github.com/Nedramrez)
+-  Twitter: [@zermarden](https://twitter.com/zermarden)
+-  LinkedIn: [LinkedIn](https://linkedin.com/in/zermarden)
 
 ## 🤝 Contributing
 
-I want to say thank you to my learning partners, morning session and standup teams!
+I want to say thank you to all my learning partners, morning session and standup teams!
```

Comment:
- This should also contain a link to the issues page of your portfolio repo on Github. please update this.